AutomatorServer/ios/AutomatorServerUITests/PatrolAutomation.swift

```diff
@@ -70,6 +52,44 @@ class PatrolAutomation {
       start.press(forDuration: 0.1, thenDragTo: end)
     }
   }
+  
+  // MARK: General UI interaction
```

Comment:
MARK?

Reply:
Yeah, it's quite nice:
<img width="933" alt="Screenshot 2022-10-01 at 5 30 26 PM" src="https://user-images.githubusercontent.com/40357511/193416716-055d3003-1642-4a9f-bd1b-0ece1df50728.png">
<img width="143" alt="Screenshot 2022-10-01 at 5 30 41 PM" src="https://user-images.githubusercontent.com/40357511/193416738-d8a10685-8359-4718-b4fc-5c396f6c93cc.png">